### PR TITLE
Severe Anxiety Culture Bottle Patch

### PR DESCRIPTION
**Pull request by unit0016**
## Why It's Good For The Game
Imagine being tastelessly edgy

## Changelog

:cl:
del: The severe anxiety culture bottle's description has been rewritten.
/:cl:

### CODE REVIEW

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -2164,9 +2164,9 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	surplus = 20
 
 /datum/uplink_item/role_restricted/anxiety
-	name = "Anxiety Culture Bottle"
-	desc = "A bottle of pure contagious autism.\
-			At least, that's what the label says"
+	name = "Severe Anxiety Culture Bottle"
+	desc = "Designed to take the concept of anxiety very literally, \
+	this culture bottle will cause the host to vomit butterflies uncontrollably."
```
Comment (nednaZ) [inline code/modules/uplink/uplink_items.dm:2169]: I do agree with the other commenter. Outright saying what the disease does removes some of the mystery.

Comment (unit0016) [reply code/modules/uplink/uplink_items.dm:2169]: I would argue noob traps are in fact bad for a server filled with new players.

Comment (nednaZ) [reply code/modules/uplink/uplink_items.dm:2169]: Virology in general is full of oddly named symptoms with no obvious effects.
SS13 is a game about discovery, so I just want at least a little mystery in the description change.

Comment (unit0016) [reply code/modules/uplink/uplink_items.dm]: desc = "If left untreated subject will regurgitate butterflies."


I have updated the description to be less blunt, but I find the entire argument silly and would almost venture to say it was started *just* to keep the original description around longer.

File: code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -2164,9 +2164,10 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	surplus = 20
 
 /datum/uplink_item/role_restricted/anxiety
-	name = "Anxiety Culture Bottle"
-	desc = "A bottle of pure contagious autism.\
-			At least, that's what the label says"
+	name = "Severe Anxiety Culture Bottle"
+	desc = "This culture bottle will cause the host to produce fully-grown \
+	specimins of Arthropoda Rhopalocera in large quantities upon injection \
```
Comment (nednaZ) [inline code/modules/uplink/uplink_items.dm:2169]:
```suggestion
	specimens of Arthropoda Rhopalocera in large quantities upon injection \
```

**Review by nednaZ — APPROVED** [review]
Perfect. I like the new description.

File: code/modules/uplink/uplink_items.dm (hunk 1 above)
Comment (nednaZ) [reply code/modules/uplink/uplink_items.dm]: This is my favorite symptom/disease, so obviously I want a good description.
I'm entirely open with replacing old code/descriptions with better ones so long as the replacement is better than what was originally there.

**Pull request merged**